lib/libzutil/zutil_import.c

```diff
@@ -993,7 +993,7 @@ zpool_read_label_slow(int fd, nvlist_t **config, int *num_labels)
 	if (num_labels != NULL)
 		*num_labels = count;
 
-	free(label);
+	posix_memalign_free(label);
```

Comment:
Seeing posix_memalign_free is confusing as I said previously. Let us just make the allocation use umem_alloc_aligned and call umem_free_aligned here instead.

Reply:
done